package.json

```diff
@@ -2,35 +2,68 @@
 	"name": "lisk-js",
 	"version": "0.4.5",
 	"description": "JavaScript library for sending Lisk transactions from the client or server",
-	"main": "./dist-node/index.js",
+	"author": "Lisk Foundation <admin@lisk.io>, lightcurve GmbH <admin@lightcurve.io>",
+	"license": "GPL-3.0",
+	"keywords": [
+		"lisk",
+		"blockchain",
+		"nodejs",
+		"javascript",
+		"library"
+	],
+	"homepage": "https://github.com/LiskHQ/lisk-js#readme",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/LiskHQ/lisk-js.git"
+	},
+	"bugs": {
+		"url": "https://github.com/LiskHQ/lisk-js/issues"
+	},
+	"engines": {
+		"node": "=6.12.3",
+		"npm": "=3.10.10"
+	},
+	"main": "dist-node/index.js",
 	"scripts": {
 		"prestart": "./bin/prestart.sh",
 		"start": "./bin/start.sh",
-		"test": "nyc mocha test --recursive",
-		"test:watch": "chokidar \"./(src|test)/**/*.js\" -c \"mocha test --recursive\" --initial",
+		"clean": "rm -rf ./node_modules ./dist-browser/lisk-js.js ./dist-browser/lisk-js.min.js ./dist-node",
```

Comment:
this was not meant to be included in the commit.
we had another command to clean up before building, but that became another script, and this got left.

However I think it's useful to have it because when changing the branch, it doesnt rsync the node_module folder.
Therefore, if you change the dependencies (especially delete), you should test it with clean install, thats why I prefer to have command to clean every non-managed stuff.